### PR TITLE
Improve06

### PR DESCRIPTION
**Pull request by jailsonboamorte**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
 
 services:
-  mongodb:
+  mongodbtakedata:
     build: ./docker/mongodb
     container_name: ${CONTAINER_MONGO_NAME}
     restart: always
     volumes:
       - ${PATH_VOLUME_MONGO_HOST}:/data/db
+      - /etc/localtime:/etc/localtime:ro
```
[the remark this answers is not in the document]
Comment (jailsonboamorte) [reply docker-compose.yml]: I thought that when you change the  Docker to Linux mode  it will work

**Comment by jailsonboamorte** [comment]
> Why deleted file .env_app?

because it is a file personal, there are sensitive information on it

**Comment by cesarjrccorreia** [comment]
> Why deleted file .env_app?
> 
> because it is a file personal, there are sensitive information on it

It's ok.

File: docker-compose.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (cesarjrccorreia) [reply docker-compose.yml]: Ok, Later I test it.

**Pull request merged**